src/nasagibs.jl

```diff
@@ -815,4 +828,4 @@ function NASAGIBSTimeseries(variant::Symbol=:reflectance; date="2020-01-01")
     return provider
 end
 
-
+PROVIDER_DICT[NASAGIBSTimeseries] = _NASAGIBS_KEYS
```

Comment:
Can you include a comment here to explain this line? Also, github's review UI complains when the file does not end with a newline

Reply:
Yes. I will do that. This is just adding the nasagibstimeseries to the main dict of providers (as you probably know...). I will also add a new line.